docs/source/markdown/podman.1.md

```diff
@@ -78,6 +78,12 @@ Identity value resolution precedence:
  - `containers.conf`
 Remote connections use local containers.conf for default.
 
+#### **--imagestore**=*path*
+
+Path of the imagestore where images are stored.  By default, the storage library stores all the images in the graphroot but if an imagestore is provided then the storage library will store newly pulled images in the provided imagestore and keep using the graphroot for everything else. If the user is using the overlay driver then the images which were already part of graphroot will still be accessible.
+
+A common use case for this field is for the users who want to split the file-system in different parts i.e disk which stores images vs disk used by the container created by the image.
```

Comment:
This should mention the storage.conf option for this.

Reply:
Fixed.